### PR TITLE
[AAE-4011] update pull policy for image

### PR DESCRIPTION
**Pull request by SohelSaiyed**

### CODE REVIEW

File: helm/alfresco-process-infrastructure/README.md [hunk 1]
```diff
@@ -360,4 +360,4 @@ Kubernetes: `>=1.15.0-0`
 | setup-acs-script-job.loadTestData | bool | `true` |  |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.4.0](https://github.com/norwoodj/helm-docs/releases/v1.4.0)
+Autogenerated from chart metadata using [helm-docs v1.3.0](https://github.com/norwoodj/helm-docs/releases/v1.3.0)
```
Comment (ffazzini) [inline helm/alfresco-process-infrastructure/README.md:363]: 1 version lower?

File: helm/alfresco-process-infrastructure/values.yaml [hunk 2]
```diff
@@ -470,7 +470,7 @@ alfresco-deployment-service:
   image:
     repository: quay.io/alfresco/alfresco-deployment-service
     tag: develop
-    pullPolicy: Always
+    pullPolicy: IfNotPresent
```
Comment (ffazzini) [inline helm/alfresco-process-infrastructure/values.yaml:473]: this line is fine, but i don't see the env var below.

File: helm/aae.yaml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+USER-SUPPLIED VALUES:
```
Comment (ffazzini) [inline helm/aae.yaml:1]: this file should be removed

Comment (SohelSaiyed) [reply helm/aae.yaml:1]: sure I will remove that

File: helm/alfresco-process-infrastructure/README.md (hunk 1 above)
Comment (SohelSaiyed) [reply helm/alfresco-process-infrastructure/README.md]: this is something was there but the doc was not updated.

File: helm/alfresco-process-infrastructure/values.yaml [hunk 4]
```diff
@@ -481,6 +481,9 @@ alfresco-deployment-service:
     image:
       # alfresco-deployment-service.applications.image.tag -- default tag for all images used in application
       tag: develop
+      pullPolicy: IfNotPresent
```
Comment (ffazzini) [inline helm/alfresco-process-infrastructure/values.yaml:484]: this new commit is not what i was asking

Comment (ffazzini) [reply helm/alfresco-process-infrastructure/values.yaml:484]: can we have a call please?

File: helm/alfresco-process-infrastructure/values.yaml (hunk 2 above)
Comment (ffazzini) [reply helm/alfresco-process-infrastructure/values.yaml:473]: sorry this line above is for deployment-service image, let's remove it form this pr please
as we can't do that for deployment-service image but not for modeling-service and apps

File: helm/alfresco-process-infrastructure/values.yaml [hunk 5]
```diff
@@ -481,6 +481,9 @@ alfresco-deployment-service:
...
+      # alfresco-deployment-service.applications.image.pullPolicy -- pull policy for alfresco-deployment-service image
```
Comment (ffazzini) [inline helm/alfresco-process-infrastructure/values.yaml:485]: for consistency this comment goes above the line and
"pull policy for alfresco-deployment-service image"
change to
"default pull policy for all images used in application"

sounds good?

File: helm/alfresco-process-infrastructure/values.yaml [hunk 6]
```diff
@@ -481,6 +481,9 @@ alfresco-deployment-service:
...
+      # @default -- do not pull new image if already exists on the running node
```
Comment (ffazzini) [inline helm/alfresco-process-infrastructure/values.yaml:486]: we can remove this line probably, all people reviewing an helm chart pr know what pull policy is for

File: helm/alfresco-process-infrastructure/values.yaml (hunk 5 above)
Comment (SohelSaiyed) [reply helm/alfresco-process-infrastructure/values.yaml:485]: yes its updated now. 
thanks

File: helm/alfresco-process-infrastructure/values.yaml (hunk 6 above)
Comment (SohelSaiyed) [reply helm/alfresco-process-infrastructure/values.yaml:486]: ok sure

**Review by ffazzini — APPROVED** [review]

**Pull request merged**